extensions/3DTILES_metadata/README.md

```diff
@@ -0,0 +1,256 @@
+# 3DTILES_metadata Extension
+
+## Contributors
+
+Cesium
+
+## Status
+
+Draft (version 0.0.0)
+
+## Overview
+
+This extension to 3D Tiles enables the declaration of metadata classes and the assignment of metadata to the tileset. Metadata may also be assigned to layers using [3DTILES_layers]().
+
+## Concepts
+
+### Metadata
+
+Metadata refers to information about the content of the tileset - it may be applicable to different components of 3D Tiles: tileset, layers and tiles.
```

Comment:
I would avoid using the word "content" since that means something specific in 3D tiles.

Maybe something along the lines of "Metadata refers to additional information attached to parts of a tileset"?